src/middlewares/checktoken.js

```diff
@@ -0,0 +1,25 @@
+import jwt from 'jsonwebtoken';
+import errorRes from '../helpers/errorHandler';
+import Models from '../database/models';
+
+const { User } = Models;
+export default async (req, res, next) => {
+  try {
+    const token = req.headers.auth?.split(' ')[1];
+
+    if (!token) return errorRes(res, 401, 'Please login first');
+
+    const decoded = jwt.verify(token, process.env.JWT_KEY);
+
+    if (!decoded) return errorRes('Unauthorized');
```

Comment:
the return responses should be consistent. please include the status code also

Reply:
I'm no longer using this in my latest work cause I have moved to redis, so in my next PR I'm not using verify(it was for making review happen )